policy/modules/system/systemd.if

```diff
@@ -22,6 +22,7 @@ template(`systemd_domain_template',`
     init_nnp_daemon_domain($1_t)
 
     kernel_read_system_state($1_t)
+    selinux_get_enforce_mode($1_t)
```

Comment:
Move the line below, kernel is the only exception to alphabetical order when using external interfaces.